src/Model/Model.java

```diff
@@ -301,15 +242,133 @@ public void addAllAddableGamesToCommandLists(String userName) {
         }
     }
 
+    public EndTurnResult sendAdvanceTurnCommands(String endTurnPlayer){
```

Comment:
You don't have to change this now, but I think this function should be split up. It violates the SRP as is, with sending back the Result, and sending commands to each player.